test/mocks.js

```diff
@@ -174,7 +174,8 @@ function spyLog (methods) {
 function mockRequest (data) {
   return {
     app: {
-      acceptLangage: 'en-US'
+      acceptLangage: 'en-US',
```

Comment:
I realize this is from previous code, but I don't speak this Langage. /cc @philbooth d